engines/sword1/sworddefs.h

```diff
@@ -138,6 +138,12 @@ struct WalkGridHeader {
 	int32 numNodes;
 } PACKED_STRUCT;
 
+struct Sprite {
+	Header header;
+	int32 totalSprites;
+	uint32 spriteOffset[2];
+} PACKED_STRUCT;
```

Comment:
I am not a big fan of packed structs, but it is how it was here before...

Reply:
Yes, I did that for consistency, but if you prefer I can always move it out of the packed structs section. Let me know 🙂